module/User/src/User/Service/User.php

```diff
@@ -98,9 +98,19 @@ public function register($data)
             return null;
         }
 
+        $newUser = $this->getNewUserMapper()->getByLidnr($data['lidnr']);
+        if (is_null($newUser)) {
+            $time = $newUser->getTime();
+            $requiredInterval = (new DateTime())->sub(new DateInterval('P1D'));
+            if ($time > $requiredInterval) {
+                $form->setError(RegisterForm::)
```

Comment:
This seems a bit weird? And with syntax errors even?

Reply:
Apparently I forgot to fill it in after creating the error.